doc/sphinx/source/input.rst

```diff
@@ -43,7 +43,7 @@ users can now continue with :ref:`Running ESMValTool <running>`.
 Observations
 ============
 
-Observational and reanalysis products in the standard CF/CMOR format used in CMIP and required by the ESMValTool are available via the obs4mips (https://esgf-node.llnl.gov/projects/obs4mips/) and ana4mips (https://esgf.nccs.nasa.gov/projects/ana4mips/) projects, respectively. Their use is strongly recommended, when possible.
+Observational and reanalysis products in the standard CF/CMOR format used in CMIP and required by the ESMValTool are available via the obs4mips and ana4mips projects at the ESGF (e.g., https://esgf-data.dkrz.de/projects/esgf-dkrz/). Their use is strongly recommended, when possible.
```

Comment:
that is not a dead link, it might have been temporarily offline but that **is** the OEM obs4mips link

Reply:
That's why I would give the overall ESGF link for both. If you search `ana4mips` or `obs4mips` you get all available datasets.

Reply:
true that, I didn't bother looking at the esgf link :grin: